compiler/example/datamodel/config/gns/gns.go

```diff
@@ -6,7 +6,7 @@ import (
 	cartv1 "github.com/vmware-tanzu/cartographer/pkg/apis/v1alpha1"
 
 	service_group "github.com/vmware-tanzu/graph-framework-for-microservices/compiler/example/datamodel/config/gns/service-group"
-	policypkg "github.com/vmware-tanzu/graph-framework-for-microservices/compiler/example/datamodel/config/policy"
```

Comment:
is it required to change this import name? if import wouldn't be changed would this work?

Reply:
yes, it should work
addressed